code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -129,6 +129,13 @@ GLOBAL_LIST_INIT(nonreactive_gases, typecacheof(list(/datum/gas/oxygen, /datum/g
 	name = "Pluoxium"
 	fusion_power = 10
 
+/datum/gas/miasma
+	id = "miasma"
+	specific_heat = 0.01
```

Comment:
*laughs in toxins*

Reply:
i like the idea of toxins using dead corpses for bomb fuel

Reply:
The low specific heat would make it a pretty good gas for fusion if you could harvest a decent amount but i feel like the work obtaining it would justify it.

Reply:
Well if you can get a experimental cloner it would be kind of easy to accumulate the amount of bodies needed for such a thing, just create a closed area without air except a scrubber set to scrub miasma set up around the cloning thing so everytime someone pops out of the pod they suffocate to death and add to the miasma generation you have. That or just humanize a bunch of monkeys and kill them and do the same set up with a enclosed space.

Reply:
![image](https://user-images.githubusercontent.com/19853115/44884768-30d03200-ac72-11e8-9cdf-19b85152d459.png)
100% balanced